build_info/base.py

```diff
@@ -21,14 +21,16 @@
 # -*- coding: utf-8 -*-
 """Classes for building external prerequisite components"""
 
-# pylint: disable=exec-used
+from __future__ import print_function
+
+#pylint: disable=exec-used
```

Comment:
Just remove it rather than commenting it out?

Reply:
Hmm, that one was accidental but not needed anymore